lego/apps/surveys/views.py

```diff
@@ -12,8 +13,10 @@
 
 
 class SurveyViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
-    queryset = Survey.objects.all().prefetch_related('questions', 'submissions')
+    queryset = Survey.objects.all().prefetch_related('questions', 'submissions'
+                                                     ).exclude(template_type__isnull=False)
```

Comment:
why does it exclude surveys without templates here?

Reply:
A survey having a `template_type` means that it _is_ a template, not that it's based on one. The idea was that fetching regular surveys and fetching templates were two separate things. However, this exclude means that templates cannot be seamlessly treated as regular surveys when you want to edit or delete or whatever, which was the motivation for not just making a separate Template model. So I'll remove this. I have to filter away templates in the survey list view front end anyway.